packages/manager/apps/web/client/app/hosting/cdn/shared/leave-confirm-settings/leave/hosting-shared-leave-settings.html

```diff
@@ -1,91 +1,80 @@
+        ></dd>
+        <dt><strong>HTTP/2</strong></dt>
+        <dd
+            data-translate="{{:: 'hosting_cdn_shared_state_' + ($ctrl.model.http.enabled ? 'enable' : 'disabled') }}"
+        ></dd>
+        <dt><strong>Devmode</strong></dt>
+        <dd
+            data-translate="{{:: 'hosting_cdn_shared_state_' + ($ctrl.model.devmode.enabled ? 'enable' : 'disabled') }}"
+        ></dd>
+        <dt><strong>Brotli</strong></dt>
+        <dd
+            data-translate="{{:: 'hosting_cdn_shared_state_' + ($ctrl.model.brotli.enabled ? 'enable' : 'disabled') }}"
+        ></dd>
+        <dt><strong>Cache rule</strong></dt>
+        <dd
+            data-ng-repeat="rule in ::$ctrl.model.rules track by $index"
+            data-ng-bind="::rule.name"
+        ></dd>
+        <div data-ng-if="$ctrl.model.cors">
+            <dt><strong>CORS</strong></dt>
+            <dd>
+                <span
+                    data-ng-if="!$ctrl.model.cors.enabled"
+                    data-translate="hosting_cdn_shared_state_disabled"
+                ></span>
+                <span
+                    data-ng-if="$ctrl.model.cors.enabled"
+                    data-ng-repeat="origin in ::$ctrl.constructor.getCorsOriginsList($ctrl.model.cors.config.origins) track by origin"
+                >
+                    <span data-ng-bind="::origin"></span>
+                    <span data-ng-if="!$last">, </span>
+                </span>
+            </dd>
         </div>
-    </div>
-
-    <!--HTTP/2-->
-    <div class="mb-2">
-        <div class="row">
-            <div class="text-right col-md-6">
-                <p class="oui-heading_5 mb-1"><strong>HTTP/2</strong></p>
-            </div>
-            <div class="text-left col-md-6">
-                <p
-                    class="mb-1"
-                    data-ng-bind="::'hosting_cdn_shared_state_' + ($ctrl.model.http.enabled ? 'enable' : 'disabled') | translate"
-                ></p>
-            </div>
+        <div data-ng-if="$ctrl.model.https_redirect">
+            <dt><strong>HTTPS-Redirect</strong></dt>
+            <dd
+                data-ng-bind="$ctrl.model.https_redirect.enabled ? $ctrl.model.https_redirect.config.statusCode : ('hosting_cdn_shared_state_disabled' | translate )"
+            ></dd>
         </div>
-    </div>
-
-    <!--DEV-MODE-->
-    <div class="mb-2">
-        <div class="row">
-            <div class="text-right col-md-6">
-                <p class="oui-heading_5 mb-1"><strong>Dev-mode</strong></p>
-            </div>
-            <div class="text-left col-md-6">
-                <p
-                    class="mb-1"
-                    data-ng-bind="::'hosting_cdn_shared_state_' + ($ctrl.model.devmode.enabled ? 'enable' : 'disabled') | translate"
-                ></p>
-            </div>
+        <div data-ng-if="$ctrl.model.hsts">
+            <dt><strong>HSTS</strong></dt>
```

Comment:
do you think it should come from a constant files?

Reply:
No this is just a label, which won't be used in javascript